docs/traffic-policy/concepts/phases.mdx

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 1
+title: Phases
+pagination_next: traffic-policy/concepts/phase-rules
+pagination_label: Phase Rules
+---
+
+# Phases
+
+In Traffic Policy, a phase represents a distinct point in the lifecycle of a request as it moves through an ngrok endpoint. Phases allow you to inspect, process, and manage traffic at key moments.
```

Comment:
```suggestion
In Traffic Policy, a phase represents a specific step in the lifecycle of a request as it moves through an ngrok endpoint. Phases allow you to inspect, process, and manage traffic at key moments.
```

Reply:
do we want to say something about how they roughly map to the phases of an http session as defined here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Session

also, maybe something about all traffic is not guaranteed to have all phases. For example, raw TCP traffic will not have any HTTP phases.

Reply:
oh, i guess you have it later. oops